quickwit/quickwit-serve/src/lib.rs

```diff
@@ -967,7 +970,7 @@ async fn setup_control_plane(
         default_index_root_uri,
         replication_factor,
     };
-    let (control_plane_mailbox, _control_plane_handle) = ControlPlane::spawn(
+    let (cp_mailbox, _cp_handle, mut readiness_rx) = ControlPlane::spawn(
```

Comment:
I personally preferred the longer name...

Now we have `cp` or `control_plane_` depending on the file in we are in in our code base.

Reply:
fixed